src/app/ApplicationTemplate.Shared.Views/Configuration/LocalizationConfiguration.cs

```diff
@@ -187,6 +188,9 @@ public bool TryApply()
 				culture = _fallbackCulture;
 			}
 
+			// Set the language override. This is necessary on WinUI for the ResourceLoader to work.
```

Comment:
Is it only required on Windows? Does it have any impacts on mobile? May be we should add `#if __WINDOWS__` for it?